src/main/scala/scala/tools/refactoring/analysis/ImportsToolbox.scala

```diff
@@ -30,6 +33,34 @@ class IsNotInImports[C <: CompilationUnitDependencies with common.EnrichedTrees]
       owners
     }
 
+    /** Returns `true` if import has been found for tested `Select` and `false` otherwise.
```

Comment:
Replace "if import" with "if an import"; apart from that, can you clarify where the tested select is in the examples below (my guess would be `b`, right?)